src/patterns-hmrc-govuk-team/notices/index.njk

```diff
@@ -55,9 +50,11 @@ Do not include:
 
 <p class="govuk-body">For VAT Notices, the title should be‘[Description] (VAT Notice [number])’.</p>
 
-<h4 class="govuk-heading-s">Details</h4>
+<h4 class="govuk-heading-s">Body</h4>
+
+<p class="govuk-body">Enter the contents of the notice. Do not include numbers in section headings.</p>
 
-<p class="govuk-body">Add this text to the end of the guide.</p>
+<p class="govuk-body">Then add this text to the end of the guide.</p>
```

Comment:
With this kind of wording (the 'then' bit) I'm tempted to suggest using `<ol>`. It could be styled or unstyled to look like an ordered list. I think there's a few of these kinds of things in the HMRC GOV.UK docs?